testing/e2e/git_rules_testkit.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Title        git_rules_testkit.sh
+# Description  Test kit for glueing together Bazel wiring and git_cached repo rules
+#==================================================================================
+
+source ./testing/e2e/bazel.sh
+
+function recreate_git_cached_rule_for_small_repo() {
+  # Re-create WORKSPACE file and re-declare git_cached_repository with the new commit hash
+  create_workspace_file_targeting_wix_build_tools "${WORKSPACE_DIR}/WORKSPACE"
```

Comment:
Again - what is `WORKSPACE_DIR` and how does it gets assigned?

Reply:
You might have missed the [previous comment](https://github.com/wix-playground/wix_build_tools/pull/4#discussion_r621070505) regarding the `WORKSPACE_DIR`.

> what is WORKSPACE_DIR

It is an exported variable created by the testing framework assigned with the temporary workspace directory path per test use case.

> how does it gets assigned

It is being created and assigned for every test case as can be seen [in here](https://github.com/wix-playground/wix_build_tools/blob/aaf49d790ae0efc5c7a2f1c2e1442e6aa0a01b09/testing/e2e/bazel.sh#L27)